hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/upgrade/TestOMVersionManager.java

```diff
@@ -101,6 +103,7 @@ public void testOMLayoutFeaturesHaveIncreasingLayoutVersion()
     verify(omMock, times(UpgradeActionType.values().length)).getVersion();
   }
 
+  @Ignore
```

Comment:
What is the reason for ignoring this test now? Maybe we should add a comment explaining why it is ignored.

Reply:
Added comment.